### ISSUE TITLE
Xcode 16.0 beta issue: Ambiguous type name 'Layout' in 'Text'

### ISSUE THREAD

**Issue by LucasVanDongen** [nathantannar4/Turbocharger#6]
```
Ambiguous type name 'Layout' in 'Text'
Found candidate with type 'Text.Layout'
Found candidate with type 'Text.Layout' (SwiftUICore.Text)
```

I guess you should rename the internal type Layout to something else?

**Comment by edwardsanchez** [comment]
+ 1

**Comment by nathantannar4** [comment]
Fixed in `1.3.0`
